src/locales/en.ts

```diff
@@ -73,7 +73,20 @@ export const en: LocaleType = {
     messenger: 'Messenger',
     search: 'Search',
   },
-
```

Comment:
feature profile , settings obj in profile

Reply:
ok